本文フォントサイズを可変に（ViewerのUIで変更可能に）

MurakamiShinyu opened the pull request.

MurakamiShinyu committed:
本文フォントサイズを可変に（ViewerのUIで変更可能に）

spring-raining commented on theme/scss/base.scss:
```diff
@@ -3,7 +3,6 @@
 :root {
   font-family: $font-family;
   font-weight: 400;
-  font-size: 16px;
```
この項目ですが、書いた当時はルート要素にフォントサイズを指定してあげないと治らなった問題があり、その問題を解決するために明示的にfont-sizeを追記した記憶があります (すみませんが問題の内容は思い出せていないです)。この状態で問題が再発しないか、後で見てみます

MurakamiShinyu replied:
https://github.com/vivliostyle/vivliostyle.js/issues/549
このバグにより、table内のフォントサイズが不正になる問題がありました。
その対策を入れました。
（table内はfont-size固定に）

MurakamiShinyu committed:
table内のフォントサイズが正しくなくなる不具合に対処

https://github.com/vivliostyle/vivliostyle.js/issues/549
このバグにより、table内のフォントサイズが正しくなくなる不具合がありました。

（:root の％値のfont-sizeがtable内で％の倍率が二重に適用されるバグがあって、16px x 75% x 75% x （tableに指定された）80% = 7.2px というフォントサイズになってしまう）

table に計算済みのfont-size値を指定することで対処
